CONTRIBUTING.md

```diff
@@ -114,6 +114,16 @@ cd api-bench
 npm run coai
 ```
 
+#### Running Tests Locally
+
+To see a list of available CI tasks and their corresponding commands to run them locally, you can run `npm run local` in the source folder. This is particularly useful for testing specific CI tasks before committing your changes.
```

Comment:
we usually add a line break after 80 characters to make it unnecessary to scroll